### ISSUE TITLE
Firebase Deployment

### ISSUE THREAD

**Issue by KillerBee05** [r-park/todo-react-redux#68]
Within the firebase.json. The public is set to "target". When I deploy it doesn't appear to find this path or directory. This is a recent issue. I've deployed fine before. Now all of the sudden it can't find the directory or path. Are you still able to deploy or are you having this issue ?

**Comment by KillerBee05** [comment]
Also npm run build fails even when I started from scratch. Npm start works as well as firebase serve.

**Comment by r-park** [comment]
Hi @KillerBee05 – I was able to clone, build, and deploy. What kind of errors are you seeing?

**Comment by KillerBee05** [comment]
Thank you for taking your time to check. Within the Command Prompt I get cannot find public directory. So it can't find the specified "target" within the firebase.json. I change 'target' to 'src' and it deploys just fine, just can't see anything on the page. I change it back to target and I get this error. I tried cloning your repo and starting over, and I ran into the same issue. I updated my firebase tools recently. Im not sure if thats the issue though. 

![image](https://cloud.githubusercontent.com/assets/21200780/25560397/b444504c-2d18-11e7-8899-cc39f5f1aea3.png)

**Comment by KillerBee05** [comment]
![image](https://cloud.githubusercontent.com/assets/21200780/25560408/49f22ace-2d19-11e7-9f28-63d7ef617854.png)

**Comment by r-park** [comment]
You mentioned that the build was failing as well — what errors are you seeing?

`target` directory is created by `npm run build`, so if the build is failing then the directory and build artifacts are probably not being generated.

**Comment by KillerBee05** [comment]
Oh okay. This is what im getting when I run the build.
![image](https://cloud.githubusercontent.com/assets/21200780/25560583/d0850efe-2d1d-11e7-8110-47a7244789e3.png)

**Comment by r-park** [comment]
Looks like eslint may have found some errors. Try running `npm run lint` and see if there are any problems reported by eslint. 

An error would look something like this:

```shell
$ npm run lint

> todo-react-redux@0.6.0 lint /Users/rpark/todo-react-redux
> eslint -c .eslintrc.yml *.js src server

/Users/rpark/todo-react-redux/karma.conf.js
  19:5  error  Parsing error: Unexpected token, expected ,

✖ 1 problem (1 error, 0 warnings)
```

**Comment by KillerBee05** [comment]
Yeah that was it, holy cow that was a pain to clear out some these errors. 

![image](https://cloud.githubusercontent.com/assets/21200780/25564874/03e2fae0-2d82-11e7-8413-73fdfce09c3d.png)

I ended up having to comment these lines out cause I don't know what it wants. I've tried everything to get them to go away. Any pointers to get it Lint to be satisfied?

![image](https://cloud.githubusercontent.com/assets/21200780/25564889/5070595c-2d82-11e7-8451-4d6c2bfcb776.png)
